CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+CHANGES IN VERSION 1.6.6-beta (from 1.6.5)
+===================================
+
+**Note:** This is a **_BETA_** release. Please direct any issues to the [Appium
+issue tracker](https://github.com/appium/appium/issues) and provide as much
+information as possible.
+
+This release exists to provide an updatable package in order to get the latest
+work on Appium. To install, first uninstall Appium (`npm uninstall -g appium`)
```

Comment:
We could also mention Xcode cache cleanup and old WDA removal from real device somewhere